src/coexistence/service.clj

```diff
@@ -169,6 +172,7 @@
 
 (def common-interceptors (into component-interceptors [(body-params/body-params) http/html-body]))
 
+;;HERE the interesting routes are "/add-address2" :get and :post , that is the base case on which I could build the rest. I just followed the pattern. Need to understand what the interceptors actually do the db instance
```

Comment:
which interceptors  for the "/add-address2" :post route?

Reply:
Please formulate the question completely, I don't know what you're trying to accomplish